console/src/components/Services/Data/TableModify/ModifyTable.js

```diff
@@ -221,6 +221,69 @@ const ColumnEditor = ({
       </option>
     );
   }
+
+  const generateAlterOptions = datatypeOptions => {
+    return dataTypes.map((datatype, index) => {
+      console.log(datatype, datatypeOptions);
+      if (datatypeOptions.includes(datatype.value)) {
+        return (
+          <option
+            value={datatype.value}
+            key={index}
```

Comment:
I used `index` as `key` because it is being used in alterTypeOptions in line 43
https://github.com/hasura/graphql-engine/blob/5d1dab6615efdbec352c7af81e9405205788dc9c/console/src/components/Services/Data/TableModify/ModifyTable.js#L43

Should I change it?

Reply:
it's anti-pattern:
https://vasanthk.gitbooks.io/react-bits/anti-patterns/06.using-indexes-as-key.html
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318